pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -143,12 +144,12 @@ func (s *Service) getOrCreateListener(openStackCluster *infrav1.OpenStackCluster
 	}
 
 	if err := s.waitForLoadBalancerActive(lbID); err != nil {
-		record.Warnf(openStackCluster, "FailedCreateListener", "Failed to create listener %s: %v", listenerName, err)
+		record.Warnf(openStackCluster, "FailedWaitForLoadBalancerActive", "Failed to wait for load balancer active %s: %v", lbID, err)
```

Comment:
I think we should also add the name here for consistency (like in the event above)